cp_businessdiary/models/business_diary.py

```diff
@@ -62,7 +62,7 @@ class BusinessDiary(models.Model):
 
     @api.multi
     @api.onchange('name')
-    def dis_pic(self):
-        ext = self.env['res.partner'].browse(self.name.id)
-        self.photo = ext.image
-        self.email = ext.email
+    def display_image(self):
```

Comment:
did you also change from where the method is called?

Reply:
you removed self.email. do not we need it anymore?

Reply:
hello @JayVora-SerpentCS 
email was already related field.so we don't need to add it again in onchange method.